flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/SqlToOperationConverter.java

```diff
@@ -1545,6 +1548,23 @@ private Operation convertDelete(SqlDelete sqlDelete) {
                 contextResolvedTable, queryOperation, SinkModifyOperation.ModifyType.DELETE);
     }
 
+    private Operation convertUpdate(SqlUpdate sqlUpdate) {
+        // set it's update
+        RowLevelModificationContextUtils.setModificationType(
+                SupportsRowLevelModificationScan.RowLevelModificationType.UPDATE);
+        RelRoot updateRelational = flinkPlanner.rel(sqlUpdate);
+        // get target sink table
+        LogicalTableModify tableModify = (LogicalTableModify) updateRelational.rel;
+        List<String> targetTablePath = tableModify.getTable().getQualifiedName();
```

Comment:
nit: use `UnresolvedIdentifier` directly to keep consistent with the delete operation